pkg/api/schemas/language.go

```diff
@@ -8,12 +8,26 @@ type UnifiedChatRequest struct {
 
 // UnifiedChatResponse defines Glide's Chat Response Schema unified across all language models
 type UnifiedChatResponse struct {
-	ID      string        `json:"id,omitempty"`
-	Created float64       `json:"created,omitempty"`
-	Choices []*ChatChoice `json:"choices,omitempty"`
-	Model   string        `json:"model,omitempty"`
-	Object  string        `json:"object,omitempty"` // TODO: what does this mean "Object"?
-	Usage   Usage         `json:"usage,omitempty"`
+	ID               string           `json:"id,omitempty"`
+	Created          float64          `json:"created,omitempty"`
+	Provider         string           `json:"provider,omitempty"`
+	Router           string           `json:"router,omitempty"`
+	Model            string           `json:"model,omitempty"`
+	Cached           bool             `json:"cached,omitempty"`
+	ProviderResponse ProviderResponse `json:"provider_response,omitempty"`
```

Comment:
Also, we may want to camelCase our JSON responses. It's somewhat popular in REST but I definitely see services using snake_case (and all sorts of cases really, just need to be consistent across our API)

Reply:
We can use camelCase